app/assets/javascripts/modules/track-qa-choices.js

```diff
@@ -0,0 +1,30 @@
+window.GOVUK = window.GOVUK || {}
```

Comment:
Does this file need including or referencing in the template somewhere, or does that happen magically?

Reply:
I believe https://github.com/alphagov/finder-frontend/blob/master/app/assets/javascripts/application.js#L19 includes it in the asset pipeline so it's compressed into `application.js`